package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "hackmud_wiki",
+  "version": "0.0.1",
+  "description": "Knowledge repository for the hackmud video game",
+  "author": "ComCODE",
+  "license": "CC-BY-NC-SA-4.0",
+  "private": true,
+  "scripts": {
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "lint": "prettier --check .",
+    "fix": "prettier --write ."
+  },
+  "dependencies": {
+    "@docusaurus/core": "^2.4.3",
+    "@docusaurus/plugin-content-docs": "^2.4.3",
+    "@docusaurus/theme-classic": "^2.4.3"
+  },
+  "devDependencies": {
+    "@docusaurus/module-type-aliases": "^2.4.3",
+    "@tsconfig/docusaurus": "^2.0.1",
+    "prettier": "^3.0.3",
+    "typescript": "^5.2.2"
```

Comment:
remove the leading ^ on typescript. We don't want this version to upgrade accidentally with lockfile shenanigans, and being explicit here is more accurate anyhow.

Reply:
Fixed in https://github.com/comcode-org/hackmud_wiki/pull/2/commits/cd8359348c4cfd367f1230d227b5f25c5d8befae